plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -161,11 +161,20 @@ public void start(boolean downloadUc) {
         System.out.println("Done");
     }
 
+    /**
+     * Create plugin directory
+     */
     void createRefDir() {
+
+        if ((refDir == null) ||
```

Comment:
I think these changes were meant for a different PR?
you should create a new branch for this and create a PR from there